src/main/java/Duke.java

```diff
@@ -1,39 +1,57 @@
 import java.util.Scanner;
+import java.util.ArrayList;
 
 public class Duke {
     private static String line = "__________________________________________________________";
+    private static Task[] inputList = new Task[101];
+    private static int length = 0;
 
-    public static void addList() {
-        Scanner in = new Scanner(System.in);
+    public static void addTask(String userInput) {
+        Task t;
+        if (userInput.contains("todo")) {
+            t = new Todo(userInput.substring(userInput.indexOf(" "), userInput.length()));
+        } else if (userInput.contains("deadline")){
+            t = new Deadline(userInput.substring(userInput.indexOf(" "), userInput.length()));
+        } else {
+            t = new Event(userInput.substring(userInput.indexOf(" "), userInput.length()));
+        }
+        t.printAddTask();
+        inputList[length+1] = t; //1-index
+        length++;
+        System.out.println("Now you have " + length + " tasks in the list.");
+        System.out.println(line);
+    }
 
-        Task[] inputList = new Task[101];
+    public static void printList(Task[] input) {
+        System.out.println(line + "\nHere are the tasks in your list: ");
+        for (int i = 1; i < input.length; i++) {
+            if (input[i] == null) {
+                break;
+            }
+            System.out.print((i) + ".");
+            input[i].printTask();
+        }
+        System.out.println(line);
+    }
+
+    public static void commands() {
+        Scanner in = new Scanner(System.in);
+        //Task[] inputList = new Task[101];
         String userInput = in.nextLine();
         boolean exit = false;
-        int length = 1;
+        //int length = 0; //0-index
         while (!exit) {
-            if (userInput.equals("bye")) {
+            if (userInput.contains("bye")) {
```

Comment:
I think equals() might be better here so in case a task has substring "bye" in it the program doesn't exit